include/alpaka/kernel/TaskKernelCpuThreads.hpp

```diff
@@ -51,11 +51,13 @@ namespace alpaka
 
     public:
         template<typename TWorkDiv>
-        ALPAKA_FN_HOST TaskKernelCpuThreads(TWorkDiv&& workDiv, TKernelFnObj const& kernelFnObj, TArgs&&... args)
+        ALPAKA_FN_HOST TaskKernelCpuThreads(
+            TWorkDiv&& workDiv,
+            KernelBundle<TKernelFnObj, TArgs...> const& KernelBundle)
             : WorkDivMembers<TDim, TIdx>(std::forward<TWorkDiv>(workDiv))
-            , m_kernelFnObj(kernelFnObj)
-            , m_args(std::forward<TArgs>(
-                  args)...) // FIXME(bgruber): this does not forward, since TArgs is not a deduced template parameter
+            , m_kernelFnObj(KernelBundle.m_kernelFn)
+            , m_args(KernelBundle.m_args) // FIXME(bgruber): this does not forward, since TArgs is not a deduced
```

Comment:
This comment should be moved to the to the constructor of kernel bundle, it now applies there.

Reply:
I removed the comment. Never the less all `TaskKernel*` will be refactored in a follow-up where we will not unpack the bundle to store function and arguments seperate because we have it already in the bundle.